packages/plugin-ext/src/main/browser/custom-editors/glob.ts

```diff
@@ -0,0 +1,742 @@
+/********************************************************************************
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+// copied from https://github.com/Microsoft/vscode/blob/bf7ac9201e7a7d01741d4e6e64b5dc9f3197d97b/src/vs/base/common/glob.ts
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as strings from '@theia/core/lib/common/strings';
+import * as paths from './paths';
+import { CharCode } from '@theia/core/lib/common/char-code';
+
+/* eslint-disable @typescript-eslint/no-shadow, no-null/no-null */
+export interface IExpression {
```

Comment:
Please add a line return here, I got confused the global eslint comment was for the interface, but it is not.

Reply:
Fixed it also in callhierarchy/src/common/glob.ts as file here is copy of that one.